state_of_html.yml

```diff
@@ -91,36 +91,36 @@ translations:
   # forms
 
   - key: features.form_controls
-    t: Form Controls
+    t: Controles de Formulário
   - key: features.form_controls.question
-    t: Which form controls have you used?
+    t: Quais controles de formulário você já utilizou?
 
   - key: features.form_input_types
-    t: Input Types
+    t: Tipos de Input
   - key: features.form_input_types.question
-    t: Which input types have you used?
+    t: Quais tipos de input você já utilizou?
   - key: options.form_input_types.na
-    t: <span aria-hidden="true">🚫</span> None of the above
+    t: <span aria-hidden="true">🚫</span> Nenhuma das opções acima
 
   - key: features.form_validation_features
-    t: Form Validation
+    t: Validação de formulário
   - key: features.form_validation_features.question
-    t: Which of the following form validation features have you used?
+    t: Quais dos seguintes recursos de validação de formulário você já utilizou?
   - key: options.form_validation_features.na
-    t: <span aria-hidden="true">🚫</span> None of the above
+    t: <span aria-hidden="true">🚫</span> Nenhuma das opções acima
 
   - key: features.forms_pain_points
-    t: Form Pain Points
+    t: Pontos problemáticos do formulário
```

Comment:
Aqui tenho dúvidas se a melhor tradução não seria "Pontos fracos..."

Reply:
tinha ficado na duvida entre essas duas traduções, mas acabei optando por pontos problemáticos

Reply:
Seria bom a opinião de mais pessoas do time, pra gente ter um melhor consenso.
@the-physicist @HakaCode @vilaboim @rodrigoant

Reply:
Talvez uma tradução literal seria melhor, **Pontos de dor**, pois reparem nesta frase _What are your pain points around HTML forms?_, está perguntando o que te incomoda, o que é difícil para você desenvolver com formulários.

Aí a tradução ficaria: **Quais são os seus pontos de dor com os formulários HTML?**
Uma palavra alternativa seria **Dificuldades**.

Reply:
Nesse caso eu acho melhor _**não**_ optar pela tradução literal porque não temos muita familiaridade com a expressão "pontos de dor". De forma análoga, "pontos problemáticos" também parece problemático (rs), mesmo que seja o sentido mais próximo do original.

Eu acredito na flexibilidade de adaptação para nosso idioma, por isso sugiro a tradução que soe mais natural. Faço as seguintes sugestões:

- Tradução como `ponto fraco`
_What are your pain points around HTML forms?_
**Quais são seus pontos fracos em relação aos formulários HTML?**

- Tradução como `ponto fraco` com mais flexibilização
_What are your pain points around HTML forms?_
**Quais são seus pontos fracos ao trabalhar com formulários em HTML?**

- Tradução como `dificuldade`
_What are your pain points around HTML forms?_
**Quais são suas dificuldades em relação aos formulários HTML?**

- Tradução como `dificuldade` com mais flexibilização
_What are your pain points around HTML forms?_
**Quais são suas dificuldades usando formulários em HTML?**

Sintam-se livre para sugerirem outras combinações!

Reply:
Muito bom as sugestões.
Acabei considerando `dificuldade` no fim por ser mais 'paint point' mesmo do form html e porque sempre temos dificuldades com o form